### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by edwinksl**
### What does this implement/fix? Explain your changes.

Fix typos in `options.json`.

### Does this close any currently open issues?

No.

### Any other comments?

No.

### Checklist

Check all those that are applicable and complete.

- [ ] Merged with latest `master` branch
- [ ] Regenerate documentation with `npm run docs`
- [ ] Add change details to `CHANGELOG.md` under "Next" section
- [ ] Added examples for testing to [examples/ directory](examples/)
- [ ] Travis CI passes (Mac support)
- [ ] AppVeyor passes (Windows support)

### CODE REVIEW

**Comment by Glavin001** [comment]
Thanks for noticing and fixing these!

The `src/options.json` is actually a generated file which is committed because users do not need to regenerate it on install (was causing issues and users were reporting failed installations). 

You will actually want to change the applicable files in https://github.com/Glavin001/atom-beautify/tree/master/src/languages

And then run `npm run docs` to regenerate the docs. Commit those changes.

Thanks again for contributing!

File: src/options.json [hunk 1]
```diff
@@ -4300,7 +4300,7 @@
           "bmatrix",
           "pmatrix"
         ],
-        "decription": "Aligns columns by the alignment tabs for environments specified",
```
Comment (Glavin001) [inline src/options.json]: How did this happen?

File: src/options.json [hunk 2]
```diff
@@ -5439,7 +5439,7 @@
         "title": "Don't join curly brackets",
         "type": "boolean",
         "default": true,
-        "descriotion": "",
```
Comment (Glavin001) [inline src/options.json]: I am extremely confused.

Comment (edwinksl) [inline src/options.json]: No idea how "p" became "o".

Comment (Glavin001) [inline src/options.json]: Found it: https://github.com/Glavin001/atom-beautify/blame/c881bdf656f6ab3daf5516209d3960f97ba1ab83/src/languages/nginx.coffee#L41

Sigh. Looks like I should add spell check along with the linter 😝 .

**Comment by edwinksl** [comment]
@Glavin001 I just found some typos in a couple of `nginx.coffee` and `python.coffee` in `src/beautifiers`. Can I directly change them or is there somewhere else I should look at?

**Comment by Glavin001** [comment]
Do you mean in `src/languages`? https://github.com/Glavin001/atom-beautify/blob/master/src/languages/nginx.coffee and https://github.com/Glavin001/atom-beautify/blob/master/src/languages/python.coffee ? You can make changes there, as I think you have already done with previous commits, and make sure to run `npm run docs` when you're finished.

**Comment by edwinksl** [comment]
My bad, I wasn't very clear about which files I was talking about.

In https://github.com/Glavin001/atom-beautify/blob/master/src/languages/nginx.coffee#L37 in `src/languages/nginx.coffee`, I believe `dontJoinCurlyBracet` is misspelled and should be `dontJoinCurlyBrackets` (I also pluralized it). This would affect https://github.com/Glavin001/atom-beautify/blob/master/src/beautifiers/nginx-beautify.coffee#L22 in `src/beautifiers/nginx-beautify.coffee`. Likewise, for Python, `formater` should be changed to `formatter` and `formaters` to `formatters` in https://github.com/Glavin001/atom-beautify/blob/master/src/languages/python.coffee#L37 and  wherever appropriate in https://github.com/Glavin001/atom-beautify/blob/master/src/beautifiers/pybeautifier.coffee.

For https://github.com/Glavin001/atom-beautify/blob/master/src/beautifiers/nginx-beautify.coffee#L22 and https://github.com/Glavin001/atom-beautify/blob/master/src/beautifiers/pybeautifier.coffee, I am guessing I can just change them directly?

**Comment by Glavin001** [comment]
Oh nooooooooo. These are actual option names meaning if they are changed and users have configured them it will suddenly and mysteriously break. Yup, I definitely need to add spell check to Pull Requests somehow since it appears I am missing all of these. This is very unfortunate.

**Comment by edwinksl** [comment]
Haha, alright then, there are no other typos in the language files as far as I can tell. Go ahead and merge whenever you are free.

**Comment by Glavin001** [comment]
@edwinksl could you instead update the `title` and `description` property values to ensure they are at least spelt correctly? Such as https://github.com/Glavin001/atom-beautify/blob/master/src/languages/python.coffee#L41

**Comment by edwinksl** [comment]
@Glavin001 Sounds good, done!

**Comment by Glavin001** [comment]
Thank you very much for contributing @edwinksl! Hopefully we can keep these typos to a minimum from here on out.

**Comment by Glavin001** [comment]
Published to v0.30.0

**Pull request merged**